### ISSUE TITLE
Can't use global .css when using modules with custom options

### ISSUE THREAD

**Issue by StarpTech** [egoist/rollup-plugin-postcss#281]
When `modules` are activated with an `object` we can't use global CSS because everything is handled as modules.
```js
postcss({
        modules: {
          generateScopedName: '[folder]__[local]',
          scopeBehaviour: 'local'
        },
        extract: true
      })
```

Input:
```js
import './simple.css';

export default function Button() {
    return <div className={styles.button}></div>
}
```
```css
.btn {
    color: red;
}
```

Output:
```css
.Button__btn___yLF_B {
    color: red;
}
```

Workaround: Don't set custom modules settings.

### Proposal:
Enable CSS Modules only on file convention or add a new option to infer the `module` mode by yourself.
https://github.com/egoist/rollup-plugin-postcss/blob/master/src/postcss-loader.js#L74

**Comment by NithinBiliya** [comment]
This is also impacting all the projects using [microbundle](https://github.com/developit/microbundle). Noticed it in [create-react-library](https://github.com/transitive-bullshit/create-react-library).

**Comment by alanrubin** [comment]
Same here, want to mix module.scss with plain .scss (global), but unable to do it.

**Comment by egoist** [comment]
fixed by #292
